onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -513,16 +511,15 @@ including arg name, arg type (contains both type and shape).)pbdoc")
             InitializeSession(sess);
           },
           R"pbdoc(Load a model saved in ONNX format.)pbdoc")
-      .def(
-          "read_bytes", [](InferenceSession* sess, const py::bytes& serializedModel) {
-            std::istringstream buffer(serializedModel);
-            auto status = sess->Load(buffer);
-            if (!status.IsOK()) {
-              throw std::runtime_error(status.ToString().c_str());
-            }
-            InitializeSession(sess);
-          },
-          R"pbdoc(Load a model serialized in ONNX format.)pbdoc")
+      .def("read_bytes", [](InferenceSession* sess, const py::bytes& serializedModel) {
+        std::istringstream buffer(serializedModel);
+        auto status = sess->Load(buffer);
+        if (!status.IsOK()) {
+          throw std::runtime_error(status.ToString().c_str());
+        }
+        InitializeSession(sess);
+      },
+           R"pbdoc(Load a model serialized in ONNX format.)pbdoc")
```

Comment:
Since you are already at it can you also update the comment on line 417 to reflect the naming changes and include l3
FROM : 
R"pbdoc(Graph optimization level for this session. 0 disables all optimizations.
Whereas 1 enables basic optimizations and 2 enables all optimizations.)pbdoc");	Whereas 1 enables basic optimizations and 2 enables all optimizations.)pbdoc");
 TO: 
R"pbdoc(Graph optimization level for this session. 0 disables all optimizations.
1 enables basic optimizations,  2 enables extended optimizations and 3 enables layout optimizations.)pbdoc");	
Thanks!

Reply:
Let's say something like "and 3 *also* enables layout optimizations".

In general, my concern with calling leve 3 ORT_ENABLE_LAYOUT is that from the naming, it is not clear that this means all other optimizations + layout ones (to me it sounds more like "just do the layout optimizations").